modules/cnf-performing-end-to-end-tests-disconnected-mode.adoc

```diff
@@ -6,6 +6,10 @@
 [id="cnf-performing-end-to-end-tests-disconnected-mode_{context}"]
 = Running latency tests in a disconnected cluster
 
+Adding the error term GIT to trigger a Vale alert.
+
+Adding this in a second commit to show only 1 further Vale review comment: lower case podman is an error.
```

Comment:
> 1 further Vale review comment

Is it possible to point out where numeral should be spelled out? ISG: In general write in words numbers that are less than 10.

Reply:
Hi Michael, this would require a potential Vale rule. If it the rule was created with a level of "error", then yes it would trigger a review comment. Good idea. The only tricky part is avoiding false positives... but worth a look.